charts/kueue/templates/components/rbac/auth_proxy_service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Any reason this is located in the rbac folder?

Reply:
Thanks for your comments!

I simply followed [config/component's directory structure](https://github.com/Gekko0114/kueue/blob/main/config/components/rbac/auth_proxy_service.yaml). I thought this would be easy for other developers who intend to modify this repository.